### PR TITLE
openning doc when no file picker found

### PR DESCRIPTION
**Pull request by Nours312**

### CODE REVIEW

**Comment by Yoric** [comment]
Please change the commit message so that it looks like "Issue #xxx - bla bla".

File: app.js [hunk 1]
```diff
@@ -109,7 +109,14 @@ filePicker.notifications.addObserver("file:open", event => {
   console.log("App", "Opening file from the file picker", event.file);
   library.open(event.file).then(book => bookViewer.view(book));
 });
-
+/**
+@todo nedd add android.html chapiter
```
Comment (Yoric) [inline app.js]: I'd prefer if you filed a bug.

File: app.js [hunk 2]
```diff
@@ -109,7 +109,14 @@ filePicker.notifications.addObserver("file:open", event => {
...
+*/
+filePicker.notifications.addObserver("file:nopicker", () => {
+  console.log("App", "nopicker");
```
Comment (Yoric) [inline app.js]: Nit: Maybe a full sentence instead of "nopicker"?

File: app.js [hunk 3]
```diff
@@ -109,7 +109,14 @@ filePicker.notifications.addObserver("file:open", event => {
...
+    library.open("samples/lector.epub").then(book => bookViewer.view(book, 
+                ("mozSetMessageHandler" in navigator) ?"firefox os.html" : 'android.html')) ;
```
Comment (Yoric) [inline app.js]: Nit: Please use the same style for `"firefox os.html"` and `'android.html'`.

**Comment by Yoric** [comment]
Almost good. Can you make the few changes above and submit the revised patch?

**Comment by Nours312** [comment]
Yes, tomorrow i'll do

**Pull request closed without merging**